src/V3Const.cpp

```diff
@@ -850,7 +850,7 @@ class ConstVisitor final : public VNVisitor {
     const AstArraySel* m_selp = nullptr;  // Current select
     const AstNode* m_scopep = nullptr;  // Current scope
     const AstAttrOf* m_attrp = nullptr;  // Current attribute
-    VDouble0 m_statBitOpReduction;  // Ops reduced in ConstBitOpTreeVisitor
+    static VDouble0 s_statBitOpReduction;  // Ops reduced in ConstBitOpTreeVisitor
```

Comment:
Can we not introduce new static data please? I have been working hard on eliminating them.

Reply:
Removed.